http/handler.go

```diff
@@ -27,6 +27,9 @@ const (
 	// not to use request forwarding
 	NoRequestForwardingHeaderName = "X-Vault-No-Request-Forwarding"
 
+	// NoCORS is the name of the header telling Vault not to use CORS handler.
+	NoCORS = "X-Vault-No-CORS"
```

Comment:
Lack of CORS knowledge here, but I don't see why disabling this would be important for CLI requests. If they ignore CORS headers, then why does returning them cause a problem?